applications/openshift/master/file_groupowner_cni_conf/rule.yml

```diff
@@ -4,7 +4,7 @@ prodtype: ocp4
 
 title: 'Verify Group Who Owns The OpenShift Container Network Interface Files'
 
-description: '{{{ describe_file_group_owner(file="/etc/cni/net.d/.*", group="root") }}}'
+description: '{{{ describe_file_group_owner(file="/etc/cni/net.d/*", group="root") }}}'
```

Comment:
when introducing this, I was explicitly asked to add the "." so it matches the regex that's actually being used.